src/plugins/ethereum/utils.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+
+import {_createIdentity, createIdentities} from "./createIdentities";
+import {parseAddress, truncateEthAddress} from "./ethAddress";
+import {nodeAddressForEthAddress} from "./ethAddressNode";
+
+export const address = {
+  parseAddress,
+  truncateEthAddress,
+  nodeAddressForEthAddress,
+};
+
+export const identity = {
+  createIdentities,
+  _createIdentity,
```

Comment:
the `_` indicates to me an attribute intended to be private. Should it be renamed?

Reply:
Yea I see no reason for it to be private, its already being exported in the file its in